src/utils/package-json.ts

```diff
@@ -64,12 +74,16 @@ function findScriptConflicts(originalScripts: any, scriptsToBeAdded: any) {
   return originalScripts ? Object.keys(scriptsToBeAdded).filter(name => Object.keys(originalScripts).includes(name)) : [];
 }
 
-async function getPackageJsonChanges(isScaffold: boolean, frontendScripts: boolean) {
+async function getPackageJsonChanges(isScaffold: boolean, frontendScripts: boolean, addCds: boolean) {
   const changes: PackageJsonChange[] = [isScaffold ? scaffoldProjectPackageJson : existingProjectPackageJson];
   if (frontendScripts) {
     changes.push(frontendTestScripts);
   }
 
+  if (addCds) {
```

Comment:
As for printing the message, something declarative might be a good idea here. It's probably too early to generalize, but given that there are things to add to the package.json for each thing that is added to the project, it might be a good idea to hold a mapping of `thing` to `packageJsonAdditions` somewhere and then just collect, merge and write it.
But we should probably wait a little longer here before generalizing.